dra-evolution/pkg/api/capacity_types.go

```diff
@@ -64,12 +64,33 @@ type ResourcePoolSpec struct {
 	// Must not have more than 128 entries.
 	Devices []Device `json:"devices,omitempty"`
 
+	// SharedAllocatable are pooled resources that are shared by all
+	// devices in the pool. This is typically used when representing a
+	// partitionable device, and need not be populated otherwise.
+	//
+	// Must not have more than 32 entries.
+	//
+	// +optional
+	// +listType=atomic
+	SharedAllocatable []ResourceCapacity `json:"sharedAllocatable,omitempty"`
```

Comment:
What about `SharedConsumable` and then just `SharedConsumed` in the device struct?

Reply:
Or (as mentioned below), even just `Sharable` and then `SharableConsumed`.

Reply:
I am flexible on the names, that works for me.